src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/on_bc_system_capability_updated_notification_from_hmi.cc

```diff
@@ -135,6 +147,46 @@ void OnBCSystemCapabilityUpdatedNotificationFromHMI::Run() {
       }
       break;
     }
+    case mobile_apis::SystemCapabilityType::VIDEO_STREAMING: {
+      if (!system_capability.keyExists(strings::video_streaming_capability)) {
+        LOG4CXX_WARN(logger_,
+                     "VideoStreamingCapability is absent in the notification. "
+                     "Notification Will be ignored");
+        return;
+      }
+      if (!(*message_)[strings::msg_params].keyExists(strings::app_id)) {
+        LOG4CXX_WARN(logger_,
+                     "Notification doesn't contain an application id. Will "
+                     "be ignored");
+        return;
+      }
+
+      const auto app_id =
+          (*message_)[strings::msg_params][strings::app_id].asUInt();
+
+      auto app = application_manager_.application(app_id);
+      if (!app) {
+        LOG4CXX_WARN(logger_,
+                     "Application with app_id: "
+                         << app_id
+                         << " isn't registered. Notification will be ignored");
+        return;
+      }
+
+      auto& system_capability_extension =
+          SystemCapabilityAppExtension::ExtractExtension(*app);
+
+      if (!system_capability_extension.IsSubscribedTo(system_capbility_type)) {
```

Comment:
@IGapchuk typo: system_capbility_type -> system_capability_type

Reply:
@LitvinenkoIra updated in 7bd8a8207b1d42cc9b6c6c94ee1c0b5028e13924